lib/std/debug.zig

```diff
@@ -336,25 +336,32 @@ pub const StackIterator = struct {
     fp: usize,
 
     pub fn init(first_address: ?usize, fp: ?usize) StackIterator {
+        if (builtin.arch == .sparcv9) {
+            // Flush all the register windows on stack.
+            asm volatile (
+                \\ flushw
+                ::: "memory");
+        }
+
```

Comment:
Would this solve the problem in an architecture-agnostic way?

```suggestion
        @fence(.Acquire);
```

Reply:
Nope, `flushw` is not a fence.